cmd/pipecd/ops.go

```diff
@@ -82,6 +84,31 @@ func (s *ops) run(ctx context.Context, t cli.Telemetry) error {
 		}
 	}()
 
+	fs, err := createFilestore(ctx, cfg, t.Logger)
+	if err != nil {
+		t.Logger.Error("failed to create filestore", zap.Error(err))
+		return err
+	}
+	defer func() {
+		if err := fs.Close(); err != nil {
+			t.Logger.Error("failed to close filestore client", zap.Error(err))
+		}
+	}()
+
+	// Set insight collector
+	{
```

Comment:
Can you add a flag (e.g. `enable-insight-collector`) to enable this feature?
Because this should be run only in dev until it got released.

Reply:
Ah, OK. I'll add the flag.